manager.go

```diff
@@ -95,12 +96,13 @@ func NewOverseer() *Overseer {
 		stateLock: &sync.Mutex{},
 	}
 
-	sigChannel := make(chan os.Signal, 2)
+	ovr.signalChan = make(chan os.Signal, 2)
 	// Catch death signals and stop all child procs on exit
-	signal.Notify(sigChannel, syscall.SIGINT, syscall.SIGTERM)
+	signal.Notify(ovr.signalChan, syscall.SIGINT, syscall.SIGTERM)
 	go func() {
 		// Catch all signals and possibly react different based on signal
-		for sig := range sigChannel {
+		// Call StopOverseer() to exit this goroutine
+		for sig := range ovr.signalChan {
 			log.Error("Received signal: %v!", Attrs{"sig": sig})
 			ovr.StopAll(false)
```

Comment:
In current implementation, calling StopAll(false) here is not needed, because StopOverseer() is required.
Maybe this logic here can be removed completely if we end up using a StopOverseer() function, this "go func" doesn't do anything special.

Reply:
OK my mistake.... this logic here auto-stops the procs when receiving SIGINT or SIGTERM. It is useful, but if we use `StopOverseer()`, this is manual mode so we don't need to listen on signals anymore, this can be done outside of Overseer at the app level.